lib/cronitor.rb

```diff
@@ -57,7 +57,7 @@ def exists?(name)
 
   def ping(type, msg = nil)
     url = "#{PING_URL}/#{code}/#{type}"
-    url += "?msg=#{URI.escape msg}" if type == 'fail' && !msg.nil?
+    url += "?msg=#{URI.escape msg}" if ['run', 'complete', 'fail'].include?(type) && !msg.nil?
```

Comment:
'run', 'complete' & 'fail' all accept messages. 'pause' doesn't according to these docs https://cronitor.io/docs/understanding-ping-urls